omod-common/src/main/java/org/openmrs/module/webservices/rest/util/ReflectionUtil.java

```diff
@@ -25,6 +31,25 @@
  */
 public class ReflectionUtil {
 	
+	private static ConcurrentMap<Pair<DelegatingResourceHandler<?>, String>, Method> setterMethodCache;

```

Comment:
Offhand I think we should clear these caches when the module activator's contextRefreshed is called.